hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -259,7 +263,14 @@ public Hedera(
                 () -> version.readableServicesVersion(),
                 () -> HapiUtils.toString(version.getHapiVersion()));
         fileServiceImpl = new FileServiceImpl();
-        contractServiceImpl = new ContractServiceImpl(instantSource);
+
+        final var appContext = new AppContextImpl(
+                instantSource,
+                new SpiSignatureVerifier(
```

Comment:
Should this be called `AppSignatureVerifier` to keep similar naming ?

Reply:
Makes sense @Neeharika-Sompalli , renamed this.